cmd/monitorcodegen/lint.go

```diff
@@ -87,5 +87,8 @@ func formatVariable(s string) (string, error) {
 		output += word
 	}
 
+	// Remove white spaces
+	output = strings.Join(strings.Fields(output), "")
```

Comment:
@jaysfx can you confirm this logic?

Reply:
Why? or How? For why, I think yaml spec allows for whitespaces for keys so we need to get rid of them. For how,  Fields() chops up the input string based whitespaces and Join() joins the bits with no space in this case.